src/server/configure.js

```diff
@@ -10,13 +10,7 @@ const environment = require('./environment')
 const publicPath = resolve(__dirname, '../../dist')
 const staticConf = { maxAge: '1y', etag: false }
 
-module.exports = (app, options = {}) => {
-  if (!options.disableRequestLogging) {
-    app.use(morgan('common'))
-  }
-  app.use(bodyParser.json())
-  app.use(cookieParser(environment.COOKIE_SECRET))
-
+function configureApiRoutes (app) {
   app.use('/api/agencies', require('./routes/agencies'))
```

Comment:
I see that the baseurl has been updated in the Vue router, but do we need any similar update for the api endpoints in the express app? Are we sure there are no collisions between this and the other GOST api urls?

Reply:
Most GOST [routes](https://github.com/usdigitalresponse/usdr-gost/blob/_staging/packages/server/src/configure.js) are namespaced under `/api/organizations/: organizationId` so this won't be an issue. The only collision is `/api/sessions` and my other PR has an import rewrite so the reference to it here will import GOST's (and the app.use call will no-op anyway since requests to that path will be handled by an earlier middleware).

Sometime after merge we may consider namespacing ARPA routes for clarity but it's not a technical necessity.